From e70eff0464b1943162c1cf4e707b4f8b9707172f Mon Sep 17 00:00:00 2001
From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
Date: Fri, 28 Mar 2014 09:53:24 +0100
Subject: Make the export-to-folder plugin less chatty.

There's no need to print when it gets activated/deacticated. None of the
other plugins do so either.

https://bugzilla.gnome.org/show_bug.cgi?id=727218

diff --git a/plugins/export-to-folder/export-to-folder.py b/plugins/export-to-folder/export-to-folder.py
index 5bb7924..e1a70f9 100644
--- plugins/export-to-folder/export-to-folder.py
+++ plugins/export-to-folder/export-to-folder.py
@@ -55,7 +55,6 @@ class ExportPlugin(GObject.Object, Eog.WindowActivatable):
         return target_dir
 
     def do_activate(self):
-        print('Activating export plugin')
         ui_manager = self.window.get_ui_manager()
         self.action_group = Gtk.ActionGroup(name='Export')
         self.action_group.add_actions([('Export', None,
@@ -64,7 +63,6 @@ class ExportPlugin(GObject.Object, Eog.WindowActivatable):
         self.ui_id = ui_manager.add_ui_from_string(ui_str)
 
     def do_deactivate(self):
-        print('Export plugin deactivated')
         ui_manager = self.window.get_ui_manager().remove_ui(self.ui_id);
 
     def export_cb(self, action, window):
-- 
cgit v0.10.1
